File: src/main/java/jp/artan/japanesefoodmod/common/potion/CommonPotion.java
package jp.artan.japanesefoodmod.common.potion;

public class CommonPotion {
    public final JapanesePotion potion;
    public final JapanesePotionType type;

    public CommonPotion(String name, boolean isBadEffectIn, int liquidColorIn) {
        this.potion = new JapanesePotion(name, isBadEffectIn, liquidColorIn);
        this.type = new JapanesePotionType(name);
    }
}

File: src/main/java/jp/artan/japanesefoodmod/common/potion/CommonPotion.kt
package jp.artan.japanesefoodmod.common.potion;

class CommonPotion(name: String, isBadEffectIn: Boolean, liquidColorIn: Int) {
    val potion = JapanesePotion(name, isBadEffectIn, liquidColorIn)
    val type = JapanesePotionType(name)
}
